src/tools/rbd/action/Group.cc

```diff
@@ -287,20 +288,19 @@ int execute_list_images(const po::variables_map &vm) {
   if (f)
     f->open_array_section("consistency_groups");
 
-  for (auto i : images) {
-    std::string image_name = i.name;
-    int64_t pool_id = i.pool;
-    int state = i.state;
+  for (auto image : images) {
+    std::string image_name = image.name;
+    int state = image.state;
     std::string state_string;
-    if (cls::rbd::GROUP_IMAGE_LINK_STATE_INCOMPLETE == state) {
+    if (GROUP_IMAGE_STATE_INCOMPLETE == state) {
       state_string = "incomplete";
     }
     if (f) {
       f->dump_string("image name", image_name);
-      f->dump_int("pool id", pool_id);
+      f->dump_int("pool", image.pool);
       f->dump_int("state", state);
     } else
-      std::cout << pool_id << "." << image_name << " " << state_string << std::endl;
+      std::cout << image.pool << "/" << image_name << " " << state_string << std::endl;
```

Comment:
@VictorDenisov  Why not present `pool_name/image_name` which i think is more readable than `pool_id/image_name`, thanks.